sfge/src/test/java/com/salesforce/rules/RuleUtilTest.java

```diff
@@ -35,16 +38,19 @@ public void setup() {
         this.g = TestUtil.getGraph();
     }
 
-    @Test
-    public void getPathEntryPoints_includesAuraEnabledMethods() {
+    @ValueSource(strings = {Schema.AURA_ENABLED, Schema.REMOTE_ACTION, Schema.NAMESPACE_ACCESSIBLE})
+    @ParameterizedTest(name = "{displayName}: {0}")
+    public void getPathEntryPoints_includesAnnotatedMethods(String annotation) {
```

Comment:
Converted the individual tests into a parameterized test that covers all three annotations.